src/data/fullness.txt

```diff
@@ -622,7 +622,7 @@ nut-shaped Crimbo cookie	2	1	awesome	7-11	0	0	0	20 Nut-Rition (+100% all stats)
 nutty organic salad	3	11	awesome	12-15	0	31-40	0	SALAD
 occult jelly donut	1	4	awesome	3-5	0	40-60	0	30 Filled With Magic (+100% spell dmg)
 oily golden mushroom	0	1	crappy	0	0	0	0	Inedible
-ojo de pez burrito	3	1	good	0	0	0	0	Unspaded 30 Fish-Eye View
+ojo de pez burrito	3	1	good	9	0	0	0
```

Comment:
30 Fish-Eye View

Reply:
no, this went away after Crimbo 2019 ended